main.cpp

```diff
@@ -258,271 +282,316 @@ struct MotorCycle
         std::string material = "Alloy";
         float currentPressure = 36.0f; // PSI
 
-        bool isWheelPressureLow(); // read whether the wheel pressure is low
-        // set wheel balance to true
+        bool isWheelPressureLow();
         void balanceWheel();
-        // read if wheels are balanced
-        bool isWheelBalanced(); // read whether or not the Wheel is balanced
+        bool isWheelBalanced();
     };
 
-    // accelerate
     void accelerate(float acceleration = 1.2f);
-    // consume gas, read tank level
-    float readGasTankLevel(); // return percentage of full tank
-    // cruise
-    bool cruise(float speed); // confirm whether or not the vehicle is cruising
-    // change Wheel
-    void getNewWheel(Wheel newWheel);
+    float readGasTankLevel();
+    bool cruise(float speed);
+    void getNewWheels(Wheel newWheel);
 
     Wheel frontWheel;
     Wheel backWheel;
 };
 
-/*
-Thing 4) Drill set
-5 properties:
-    1) drill bit material (string)
-    2) drill bit diameter (unsigned int)
-    3) angular speed (float)
-    4) number of chargers (unsigned int)
-    5) brand name (string)
-3 things it can do:
-    1) drill concrete
-    2) unscrew in the left or right direction
-    3) set rotatory speed
- */
+bool MotorCycle::Wheel::isWheelPressureLow()
+{
+    bool isPressureLow = false;
+
+    if(currentPressure < 32.0f)
+        isPressureLow = true;
+
+    return isPressureLow;
+}
+
+void MotorCycle::Wheel::balanceWheel()
+{
+    isBalanced = true;
+}
+
+bool MotorCycle::Wheel::isWheelBalanced()
+{
+    return isBalanced;
+}
+
+void MotorCycle::accelerate(float acceleration)
+{
+    revolutionsPerMinute *= acceleration;
+}
+
+float MotorCycle::readGasTankLevel()
+{
+    float percentGasLevel;
+
+    percentGasLevel = gasLevel / fullGasLevel * 100.0f;
```

Comment:
Don't declare the variable separately from where you initialize it. Do it all in one line.
```
float name = expression;
```
- [ ] revise